torch/distributed/rpc/api.py

```diff
@@ -480,6 +480,8 @@ def rpc_sync(to, func, args=None, kwargs=None):
         args (tuple): the argument tuple for the ``func`` invocation.
         kwargs (dict): is a dictionary of keyword arguments for the ``func``
                        invocation.
+        timeout (datetime.timedelta): timeout for the RPC call. If the RPC is
+                        not completed in this timeframe, an exception will be raised indicating that the RPC has timed out.
```

Comment:
shall we break this long line into short pieces?

Reply:
Should we mark this as optional and mention that by default, there is no timeout?

Reply:
Yes, this sounds good. Since I am undoing the changes in this diff, there will actually be a default timeout that is set at RPC init time.